src/node_modules/util/string.mjs

```diff
@@ -11,3 +11,16 @@ export const dedent = _.pipe([
 	_.joinWith('\n'),
 	trim
 ]);
+
+export const splitAtMost = (str, sep, n) => {
+	const prefix = _.map(range(n), () =>
+		str.slice(sep.lastIndex, sep.exec(str).index)
+	);
+	const result = [...prefix, str.slice(sep.lastIndex)];
+	return result;
+};
```

Comment:
seems unused

Reply:
I might check this code into svizzle then? Could be handy to have in future.

Reply:
let's discuss in slack!